docs/src/snforge-advanced-features/fork-testing.md

```diff
@@ -21,37 +23,44 @@ We first need to define the contract's interface along with all the structures u
 
 There are two ways of configuring a fork:
 - by specifying `url` and block-related parameters in the `#[fork(...)]` attribute
-- or by passing a fork name defined in your `Scarb.toml` to the `#[fork(...)]` attribute
+- or by providing a fork name defined in your `Scarb.toml` to the `#[fork(...)]` attribute
+
+> 📝 **Note**
+> Using fork tests means `snforge` will make (often multiple) requests to the configured RPC URL.
+> These requests are relatively slow in comparison to standard tests.
```

Comment:
Pls reword this sentence - comparison of request and test seems inadequate to me 😅

Reply:
Reworded